### DISCUSSION TITLE
When will twin macro can use with SWC instead of babel ?

### DISCUSSION THREAD

**Discussion by techvsi-huynguyen** [ben-rogerson/twin.macro#837] [Q&A]
On NextJs version 14 having server action, but we can not use this feature because babel can not compile it. I hope twin macro can change to SWC instead of Babel. So we can use this feature

**Comment by ben-rogerson** [comment]
Twin was made as a babel plugin, I've looked into a SWC rewrite and it would require a Rust based implementation which is quite a large deviation, specially if its to support a single framework. Sorry!
